app/javascript/components/search/Filters.js

```diff
@@ -38,6 +47,14 @@ class Filters extends React.Component {
     this.setState({selectedFormIds: []});
   }
 
+  handleChangeAdvancedSearch(event) {
+    this.setState({advancedSearchText: event.target.value});
+  }
+
+  handleClearFilters() {
+    submitSearch(null);
```

Comment:
When clearing search, can we remove the search query string param (and even the `?` if it's the only param) please? I'm sure there is an npm package that will do that for you!